app/Http/Controllers/API/Exports/External/CliController.php

```diff
@@ -28,15 +28,15 @@ protected function getResource()
 
     public function show(Request $request)
     {
+    	$route_params = $this->getRouteParams($request);
```

Comment:
I tried using `show(Request $request, $id)` but that didn't work :/ not sure why tbh